travis/js_tests.sh

```diff
@@ -4,5 +4,8 @@ cd open-widget-framework
 
```

Comment:
On the second line of this file can you add:

    set -e -o pipefail

This would cause the file to return a non-zero exit value  if any of the commands in the file return a non-zero exit file